src/api/qualicharge/api/v1/routers/static.py

```diff
@@ -176,18 +184,24 @@ async def read(
     ],
     session: Session = Depends(get_session),
 ) -> Statique:
-    """Read statique item (point de charge)."""
+    """Read statique item (point de charge).
+
+    Note that it can take up to 10 minutes for a created Statique item to appear in
+    this endpoint response.
+    """
     if not is_pdc_allowed_for_user(id_pdc_itinerance, user):
         raise PermissionDenied("You don't manage this point of charge")
 
     try:
-        statique = build_statique(session, id_pdc_itinerance)
-    except ObjectDoesNotExist as err:
+        statique_mv = session.exec(
+            select(StatiqueMV).where(StatiqueMV.id_pdc_itinerance == id_pdc_itinerance)
+        ).one()
+    except NoResultFound as err:
         raise HTTPException(
             status_code=status.HTTP_404_NOT_FOUND,
             detail="Requested statique does not exist",
```

Comment:
Maybe we should be more specific?

```suggestion
            detail="Requested statique does not exist yet. You should wait up to 10 minutes for a newly created entry.",
```

Reply:
The suggested change is appropriate.

There an additional note for using the API ("Note that it can take up to 10 minutes for a created Statique item to appear in this endpoint response"). 
So i think the suggested change is sufficient.